### PR TITLE
EVA-3276 - File upload

### PR DESCRIPTION
**Pull request by tcezard**

### CODE REVIEW

File: cli/submit.py [hunk 1]
```diff
@@ -38,19 +43,25 @@ def get_submission_id_and_upload_url(self, submission_dir):
         else:
             raise FileNotFoundError(f'Could not upload. No config file found for the submission in {submission_dir}.')
 
-    # TODO
-    def upload_submission(self, submission_dir, submission_id=None, submission_upload_url=None):
-        if not submission_id or not submission_upload_url:
+    def upload_submission(self, submission_upload_url, submission_dir):
```
Comment (nitin-ebi) [inline cli/submit.py:46]:
```suggestion
    def upload_submission(self, submission_dir, submission_upload_url=None):
```

**Review by nitin-ebi — APPROVED** [review]

File: cli/submit.py [hunk 2]
```diff
@@ -15,9 +18,11 @@
 
 
 class StudySubmitter:
-    def __init__(self, submission_initiate_url=SUBMISSION_INITIATE_URL):
+    def __init__(self, vcf_files, metadata_file, submission_initiate_url=SUBMISSION_INITIATE_URL):
```
Comment (apriltuesday) [inline cli/submit.py:21]: Should change [this initialisation](https://github.com/EBIvariation/eva-sub-cli/blob/17a351a25f2ad436344d16e58cbd7c40ca79ca7d/cli/eva_sub_cli.py#L16) as well... the user interaction might be a bit rough but we should get something roughly working first.

File: cli/submit.py [hunk 3]
```diff
@@ -38,19 +43,25 @@ def get_submission_id_and_upload_url(self, submission_dir):
...
 
-    # TODO
-    def upload_submission(self, submission_dir, submission_id=None, submission_upload_url=None):
-        if not submission_id or not submission_upload_url:
+    def upload_submission(self, submission_dir, submission_upload_url=None):
+        if not submission_upload_url:
             submission_id, submission_upload_url = self.get_submission_id_and_upload_url(submission_dir)
-        pass
+        for f in self.vcf_files:
+            self.upload_file(submission_upload_url, f)
```
Comment (apriltuesday) [inline cli/submit.py:50]: Maybe should log/print something for the user, as these uploads could take a while?

File: cli/submit.py (hunk 2 above)
Comment (tcezard) [reply cli/submit.py:21]: This was changing significantly in the next [PR](https://github.com/EBIvariation/eva-sub-cli/pull/14/files#diff-2eec00526c12fe9eab725e922d6e5fb4520e7769add0871942ddc1df8cedd5e4) which is why I left it as is

Comment (apriltuesday) [reply cli/submit.py:20]: Got it, I see why I was confused..

**Review by apriltuesday — APPROVED** [review]

**Pull request merged**